src/larray.jl

```diff
@@ -1,14 +1,14 @@
-struct LArray{T,N,Syms} <: DenseArray{T,N}
-  __x::Array{T,N}
-  LArray{Syms}(__x) where Syms = new{eltype(__x),ndims(__x),Syms}(__x)
-  LArray{T,N,Syms}(__x) where {T,N,Syms} = new{T,N,Syms}(__x)
+struct LArray{T,N,D<:AbstractArray{T,N},Syms} <: DenseArray{T,N}
```

Comment:
Wouldn't it make sense to be consistent and either use `D<:DenseArray{T,N}` or define `LArray{T,N,D<:AbstractArray{T,N},Syms} <: AbstractArray{T,N}`?

Reply:
Yeah that should be consistent, and I don't think it makse sense to have a labelled sparse array. But does the limitation to `DenseArray` actually have any benefit?